src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/OxoGReadCounts.java

```diff
@@ -65,55 +59,37 @@ public void annotate(final ReferenceContext refContext,
         Utils.nonNull(gb, "gb is null");
         Utils.nonNull(vc, "vc is null");
 
-        if (g == null || !g.isCalled() || likelihoods == null || !vc.isSNP()) {
+        if (g == null || likelihoods == null) {
             return;
         }
 
-        final Allele ref = vc.getReference();
-        final Allele alt = vc.getAlternateAllele(0);
-
-        int alt_F1R2 = 0;
-        int alt_F2R1 = 0;
-        int ref_F1R2 = 0;
-        int ref_F2R1 = 0;
-
-        for (final ReadLikelihoods<Allele>.BestAllele bestAllele : likelihoods.bestAlleles(g.getSampleName())) {
-            final GATKRead read = bestAllele.read;
-            if (bestAllele.isInformative() && isUsableRead(read) && read.isPaired()) {
-                final Allele allele = bestAllele.allele;
-                if (allele.equals(ref, true)) {
-                    if (read.isReverseStrand() == read.isFirstOfPair()) {
-                        ref_F2R1++;
-                    } else {
-                        ref_F1R2++;
-                    }
-                } else if (allele.equals(alt, true)) {
-                    if (read.isReverseStrand() == read.isFirstOfPair()) {
-                        alt_F2R1++;
-                    } else {
-                        alt_F1R2++;
-                    }
-                }
-            }
-        }
+        final Map<Allele, MutableInt> f1r2Counts = likelihoods.alleles().stream()
```

Comment:
I changed the annotation to do all alleles (still a genotype annotation) but left the filtering to only look at the first alt allele -- the pipeline's output should be identical.  I don't think I understand what you're getting at about FilterByOrientationBias.  Do you mean including multiallelic variants in our tests?  

Also, I agree that looking only at the first allele is a hack but I think addressing that should be in the context of a big refactoring of our filtering engine, which does similar things elsewhere.